### PR TITLE
Updates for copyright and licensing

### PR DESCRIPTION
**Pull request by eduardo-rodrigues**

### CODE REVIEW

**Comment by eduardo-rodrigues** [comment]
Totally unrelated: I also committed trivial updates to please PEP8 rules.

**Comment by eduardo-rodrigues** [comment]
BTW, to me this is the last commit before releasing 0.4.0. Indeed there are very many important updates and fixes since 0.3.0 and the open "issues" are not a massive concern.

(For completeness: issue https://github.com/scikit-hep/particle/issues/41 will require some work. I will add a few comments there shortly.)

**Comment by henryiii** [comment]
Please do not release till after I get off the train, I’m homing to add some tests on the train.

**Comment by eduardo-rodrigues** [comment]
No problem. There's a reason why I'm sending advance warning ;-). I'm just saying that a release should be imminent … I will of course wait for your green light.

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 BSD 3-Clause License
 
-Copyright (c) 2018, The Scikit-HEP Project
+Copyright (c) 2018-2019, The particle developers.
```
Comment (henryiii) [inline LICENSE]: I think The SciKit-HEP developers would be fine. (And Particle should be capitalized if used)

Comment (eduardo-rodrigues) [reply LICENSE]: That's "a bit" too much to give everyone contributing to any Scikit-HEP org package the copyright hold, I find. I will capitalise "particle", sure.

Comment (eduardo-rodrigues) [reply LICENSE]: Following the comments from Hans for boost-histogram, would you prefer to copyright to us two only, just as there? That's OK for such a well-defined package/project done 100% by us anyway.

**Comment by eduardo-rodrigues** [comment]
Shall I then change the copyright to

> Copyright (c) 2018-2019, Eduardo Rodrigues and Henry Schreiner.

or leave it as it is in this PR?

File: LICENSE (hunk 1 above)
Comment (henryiii) [inline LICENSE]: I believe we can just put our names here. I have permission to do so, anyway.

Comment (eduardo-rodrigues) [reply LICENSE]: OK, will do.

**Comment by eduardo-rodrigues** [comment]
OK, @henryiii, I adapted following what Hans and you did for boost-histogram. Ready to go if you're happy too.

**Pull request merged**